### PR TITLE
Update geo-tools and map-layers to official AppUI 4.0 release

### PR DESCRIPTION
**Pull request by mdastous-bentley**

### CODE REVIEW

**Review by aruniverse — COMMENTED** [review]
youll want to update version-policies and run rush change minor like: https://github.com/iTwin/viewer-components-react/pull/458

**Comment by mdastous-bentley** [comment]
> youll want to update version-policies and run rush change minor like: #458

Done. what's next, I guess we should make a build before upgrading core packages?

File: common/config/rush/version-policies.json [hunk 1]
```diff
@@ -61,8 +60,7 @@
   },
   {
     "policyName": "itwin-map-layers",
-    "definitionName": "lockStepVersion",
-    "version": "4.0.0-dev.51",
-    "nextBump": "prerelease"
+    "definitionName": "individualVersion",
+    "lockedMajor": 0
```
Comment (aruniverse) [inline common/config/rush/version-policies.json:64]: locked major should be 4

Comment (mdastous-bentley) [reply common/config/rush/version-policies.json]: I updated it

**Review by aruniverse — APPROVED** [review]

**Comment by mdastous-bentley** [comment]
@aruniverse looks like there is a leak in @itwin/measure-tools-react, tests are failing

**Comment by aruniverse** [comment]
> @aruniverse looks like there is a leak in @itwin/measure-tools-react, tests are failing

@Jake-Screen can you take a look?

**Pull request merged**